src/librustc_resolve/lib.rs

```diff
@@ -1654,12 +1661,17 @@ impl<'a> Resolver<'a> {
         let path: Vec<Ident> = segments.iter()
             .map(|seg| Ident::new(seg.name, span))
             .collect();
+
+        let path = ResolvePath {
+            segments: &path,
+            source: None,
+        };
```

Comment:
Nit: could you use one-line formatting for all these `ResolvePath { ... }` constructors?